### PR TITLE
accessibility attempt

### LINKED ISSUE #10: [Polish] Ensure alt-text in place for screen readers
**What needs to be done? Why does it need to be done?**
Ensure that there is alt-text usable by screen readers for all components in the entire app.

**Additional context**
Follow alt-text best practices to ensure accessibility.

### PR DESCRIPTION
**Pull request by Wmaileq**
## Description of the change

Looked through component - all controls(selects and slider) are accessible.
Tried to impress what is shown on the graph, maybe you have thoughts about it?

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #10 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/SupervisionSuccess/utils/transformChartAsText.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const transformChartDataToText = (dataset) => {
+  let result = `Initial: ${dataset[0]} people in prison.`;
+  if (dataset[12]) result += `After 1st year: ${dataset[12]}`;
```
Comment (jessex) [inline src/SupervisionSuccess/utils/transformChartAsText.js:3]: Need a space before the start of this one: ` After...`

File: src/SupervisionSuccess/utils/transformChartAsText.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  if (dataset[24]) result += ` After 2nd year: ${dataset[24]}.`;
+  if (dataset[36]) result += ` After 3rd year: ${dataset[36]}`;
```
Comment (jessex) [inline src/SupervisionSuccess/utils/transformChartAsText.js:5]: Need a period at the end of this one.

File: src/SupervisionSuccess/utils/transformChartAsText.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  if (dataset[48]) result += ` After 4th year: ${dataset[48]}`;
```
Comment (jessex) [inline src/SupervisionSuccess/utils/transformChartAsText.js:6]: Need a period at the end of this one.

File: src/SupervisionSuccess/utils/transformChartAsText.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  if (dataset[60]) result += ` After 5th year: ${dataset[60]}`;
```
Comment (jessex) [inline src/SupervisionSuccess/utils/transformChartAsText.js:7]: Need a period at the end of this one.

**Review by jessex — CHANGES_REQUESTED** [review]
A few nits on the aria-label for the chart, but I think that looks great. However, poking around the Accessibility tab in the Chrome Dev Tools, it's not obvious to me that all of the components have alt text, and I don't see other aria-labels in the codebase. How do you know those components are accessible as-is?

**Comment by Wmaileq** [comment]
> A few nits on the aria-label for the chart, but I think that looks great. However, poking around the Accessibility tab in the Chrome Dev Tools, it's not obvious to me that all of the components have alt text, and I don't see other aria-labels in the codebase. How do you know those components are accessible as-is?

I just tested it with screen reader emulator. Cmd + f5 on Mac.
The problem is if this components are not accessible we will need to find alternatives or build our custom components.
But, luckily slider and selects functions well

**Comment by jessex** [comment]
Thanks for the explanation, @Wmaileq! Once the comments are resolved, I think we're good here.

**Review by jessex — APPROVED** [review]
Good to go! 🚀

**Pull request merged**